### PR TITLE
add scripts by 222DIY

### PR DESCRIPTION
**Pull request by purerosefallen**
机巧辰-高暗御津羽靇

### CODE REVIEW

File: scripts/ROTD-JP/c101101032.lua [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+function c101101032.initial_effect(c)
+	--special summon
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_FIELD)
+	e1:SetCode(EFFECT_SPSUMMON_PROC)
+	e1:SetProperty(EFFECT_FLAG_UNCOPYABLE)
+	e1:SetRange(LOCATION_HAND)
+	e1:SetCountLimit(1,101101032)
+	e1:SetCondition(c101101032.spcon)
+	c:RegisterEffect(e1)
```
Comment (mallu11) [inline scripts/ROTD-JP/c101101032.lua:12]: the first effect is an activated effect.

File: scripts/ROTD-JP/c101101032.lua [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	if chk==0 then return Duel.IsExistingMatchingCard(c101101032.filter,tp,0,LOCATION_GRAVE,1,nil) end
+	Duel.SetOperationInfo(0,CATEGORY_REMOVE,g,1,0,LOCATION_GRAVE)
+end
+function c101101032.reop(e,tp,eg,ep,ev,re,r,rp,chk)
+	local g=Duel.GetMatchingGroup(c101101032.filter,tp,0,LOCATION_GRAVE,nil)
+	if not g then return end
+	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_REMOVE)
+	local tc=g:Select(tp,1,1,nil):GetFirst()
+	local atk=tc:GetTextAttack()
+	if tc and Duel.Remove(tc,POS_FACEUP,REASON_EFFECT)~=0 and atk>0 then
```
Comment (mallu11) [inline scripts/ROTD-JP/c101101032.lua:102]: missing `Duel.BreakEffect()`.

**Comment by mallu11** [comment]
Is there any other people scripting the script now? We have the plan to script the missing scripts and need more scripters to help.If he can join the discord to communicate, we can know which scripts he is working on.

File: scripts/ROTD-JP/c101101032.lua [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	e1:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_CONTINUOUS)
+	e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE)
+	e1:SetCode(EVENT_ATTACK_ANNOUNCE)
+	e1:SetReset(RESET_PHASE+PHASE_END)
+	e1:SetOperation(c101101032.checkop)
+	Duel.RegisterEffect(e1,tp)
+	--cannot announce
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_FIELD)
+	e2:SetCode(EFFECT_CANNOT_ATTACK_ANNOUNCE)
```
Comment (mallu11) [inline scripts/ROTD-JP/c101101032.lua:71]: I think here should use `EFFECT_CANNOT_ATTACK` and `EFFECT_FLAG_IGNORE_IMMUNE`.

**Comment by purerosefallen** [comment]
They are in my custom card community. But they are unable to access Discord.

**Pull request merged**